zoo/auditing/runner.py

```diff
@@ -14,13 +14,18 @@
 Result.__new__.__defaults__ = (None,)
 
 Patch = namedtuple("Patch", ["action", "file_path", "content", "previous_path"])
-# set default to `None` for `previous_path`
-Patch.__new__.__defaults__ = (None,)
+# set default to `None` for `previous_path` and `content`
+Patch.__new__.__defaults__ = (None, None)
+
+RequestPatch = namedtuple("RequestPatch", ["url", "method", "headers", "body"])
+# set default to `None` for `headers` and `body`
+RequestPatch.__new__.__defaults__ = (None, None)
 
 
 class CheckContext:
     Result = Result
     Patch = Patch
```

Comment:
Would it be a big deal to rename the simple `Patch` to `CodePatch`? Right now is not really clear what's the difference, and I could expect `RequestPatch` to be extending `Patch`

Reply:
Renamed, we just need to rename this also in the checks (I will do it) and merge to master at the same time.